src/rsz/test/repair_setup4_verbose.ok

```diff
@@ -0,0 +1,23 @@
+[INFO ODB-0222] Reading LEF file: Nangate45/Nangate45.lef
+[INFO ODB-0223]     Created 22 technology layers
+[INFO ODB-0224]     Created 27 technology vias
+[INFO ODB-0225]     Created 135 library cells
+[INFO ODB-0226] Finished LEF file:  Nangate45/Nangate45.lef
+[INFO ODB-0128] Design: reg1
+[INFO ODB-0130]     Created 1 pins.
+[INFO ODB-0131]     Created 17 components and 92 component-terminals.
+[INFO ODB-0132]     Created 2 special nets and 34 connections.
+[INFO ODB-0133]     Created 7 nets and 30 connections.
+worst slack -1.95
+[INFO RSZ-0094] Found 6 endpoints with setup violations.
+Iteration | Resized | Buffers | Cloned Gates | Pin Swaps |   WNS   |   TNS   | Endpoint
+---------------------------------------------------------------------------------------
+        0 |       0 |       0 |            0 |         0 |  -1.954 |  -6.294 | r2/D
+       10 |       8 |       3 |            0 |         0 |  -0.865 |  -3.085 | r2/D
+       19 |      17 |       3 |            0 |        -2 |  -0.654 |  -2.908 | r2/D
+    final |      17 |       3 |            0 |        -2 |  -0.654 |  -2.908 | r2/D
```

Comment:
Please sort out the negative values before we merge.

Reply:
I don't think we should combine what is possibly a counting error in pin swapping with a reporting change (which exposed the error). I would suggest that we merge this. I can take a look at the -ve pin swap number (will look right now).

Reply:
https://github.com/The-OpenROAD-Project/OpenROAD/pull/3630